nautobot/dcim/migrations/0060_module_models.py

```diff
@@ -0,0 +1,610 @@
+                ),
+                ("position", models.CharField(max_length=255)),
+                ("label", models.CharField(blank=True, max_length=255)),
+                ("description", models.CharField(blank=True, max_length=255)),
+                (
+                    "parent_device",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="module_bays",
+                        to="dcim.device",
+                    ),
+                ),
+                (
+                    "parent_module",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="module_bays",
+                        to="dcim.module",
+                    ),
+                ),
+                ("tags", nautobot.core.models.fields.TagsField(through="extras.TaggedItem", to="extras.Tag")),
+            ],
+            options={
+                "ordering": ("parent_device", "parent_module__id", "position"),
+            },
+            bases=(
+                nautobot.extras.models.mixins.DynamicGroupMixin,
+                nautobot.extras.models.mixins.NotesMixin,
+                models.Model,
+            ),
+        ),
+        migrations.AddField(
+            model_name="module",
+            name="module_type",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.PROTECT, related_name="modules", to="dcim.moduletype"
+            ),
+        ),
+        migrations.AddField(
+            model_name="module",
+            name="parent_module_bay",
+            field=models.OneToOneField(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="installed_module",
+                to="dcim.modulebay",
+            ),
+        ),
+        migrations.AddField(
+            model_name="module",
+            name="role",
+            field=nautobot.extras.models.roles.RoleField(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="modules",
+                to="extras.role",
+            ),
+        ),
+        migrations.AddField(
+            model_name="module",
+            name="status",
+            field=nautobot.extras.models.statuses.StatusField(
+                on_delete=django.db.models.deletion.PROTECT, related_name="modules", to="extras.status"
+            ),
+        ),
+        migrations.AddField(
+            model_name="module",
+            name="tags",
+            field=nautobot.core.models.fields.TagsField(through="extras.TaggedItem", to="extras.Tag"),
+        ),
```

Comment:
Curious why these aren't included in the `CreateModel` call above?

Reply:
I'm not sure. When I have Django recreate migrations they don't change.